src/Adapters/Storage/Postgres/PostgresStorageAdapter.js

```diff
@@ -1366,6 +1370,140 @@ export class PostgresStorageAdapter {
     });
   }
 
+  distinct(className, schema, query, fieldName) {
+    debug('distinct', className, query);
```

Comment:
Just curious what's the `debug` mark here for?

Reply:
Passing `PARSE_SERVER_LOG_LEVEL=debug`

It really help for testing PG